weblate/accounts/views.py

```diff
@@ -1345,4 +1345,8 @@ def get_context_data(self, **kwargs):
         context["sort_query"] = self.sort_query
         context["sort_name"] = self.form.sort_choices[self.sort_query.strip("-")]
         context["sort_choices"] = self.form.sort_choices
+        context["query_string"] = urlencode({
+            "q": self.form.cleaned_data.get("q", "").strip(),
+            "sort_by": self.sort_query,
+        })
```

Comment:
Easist approach to deal with thisis to install pre-commit and let it do the code formatting, see https://docs.weblate.org/en/latest/contributing/code.html#coding-standard

This should be correct:

```suggestion
        context["query_string"] = urlencode(
            {
                "q": self.form.cleaned_data.get("q", "").strip(),
                "sort_by": self.sort_query,
            }
        )
```

Reply:
Thanks @nijel . I've installed the packages as suggested at the link. I thank you for your patience.